torchgeo/datasets/potsdam.py

```diff
@@ -0,0 +1,397 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Potsdam dataset."""
+
+import os
+from typing import Any, Callable, Dict, Optional
+
+import matplotlib.pyplot as plt
+import pytorch_lightning as pl
+import rasterio
+import torch
+from matplotlib.figure import Figure
+from torch import Tensor
+from torch.utils.data import DataLoader
+from torchvision.transforms import Compose
+
+from ..datasets.utils import dataset_split, draw_semantic_segmentation_masks
+from .geo import VisionDataset
+from .utils import check_integrity, extract_archive, rgb_to_mask
+
+
+class Potsdam(VisionDataset):
+    """Potsdam dataset.
```

Comment:
Is "Potsdam" unique enough to identify this dataset and this dataset only? Or are there other Potsdam datasets (for 3D-semantic segmentation or something like that)? I think we'll want to call this Potsdam2D or maybe something even more specific.

Reply:
Changed to Potsdam2D